.github/workflows/release.yml

```diff
@@ -14,6 +14,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Check cargo package
+        # Checks if `cargo package` can run successfully. This saves us from
+        # situations where this is only noticed after a merge and releases
+        # were made only partially.
+        run: cargo package
```

Comment:
Does this work without package args? I get a failure in the `uefi_app` package when I run `cargo package` locally. `cargo package -p uefi -p uefi-raw -p uefi-macros` works, however

Reply:
I think it is fine to directly list the crates